packages/octopus-fig/src/entities/octopus/octopus-path.ts

```diff
@@ -38,6 +38,7 @@ export class OctopusPath {
       case 'RECTANGLE':
         return 'RECTANGLE'
       case 'LINE':
+      case 'VECTOR':
```

Comment:
Why `VECTOR`s are not converted to `LINE`s? 😲

Reply:
reverted